admin-notice/js/script.js

```diff
@@ -21,19 +14,18 @@ function persistDismissedNotice(containerElement) {
 	}
 }
 
-function onDismissed(event) {
-	const noticeContainer = tryGetNoticeContainer(event && event.target);
-	if (noticeContainer) {
-		persistDismissedNotice(noticeContainer);
+function vipNoticeClickHandler( ev ) {
+	const button = ev.target.closest( '.notice-dismiss' );
```

Comment:
[closest()](https://caniuse.com/element-closest) is supported by all browsers supporting [fetch()](https://caniuse.com/fetch).